.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#IntelliJ/Pycharm
+.idea
```

Comment:
I know it's more work, but it's generally better practice to keep a project's `.gitignore` file specific to files that the project itself could produce; user-generated files (like editor configurations) should really be kept in a global `.gitignore` on that specific user's computer. [This](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) is a great resource on configuring such a file.